src/mlpack/methods/logistic_regression/logistic_regression_function.hpp

```diff
@@ -82,6 +82,26 @@ class LogisticRegressionFunction
   double Evaluate(const arma::mat& parameters, const size_t i) const;
 
   /**
+   * Evaluate the logistic regression log-likelihood function with the given
+   * parameters using the given batch size from a given point.  This is useful for optimizers
```

Comment:
I'd suggest `starting from the given point index` instead of `from the given point` for clarity.